sympy/core/relational.py

```diff
@@ -1408,16 +1420,24 @@ def split_real_imag(expr):
         if n2 is not None:
             return _sympify(n2 == 0)
 
+        def is_infinite(obj, assumptions):
+            # one shot function which is faster than AssumptionsWrapper
+            if assumptions is None:
+                return obj.is_infinite
+            return ask(Q.infinite(obj), assumptions)
```

Comment:
I meant that functions like this could be defined in the same module as `AssumptionsWrapper` since they go together.

Reply:
I made assumptions/wrapper.py to define them.